pkg/operator/ceph/csi/operator_driver.go

```diff
@@ -254,6 +254,19 @@ func (r *ReconcileCSI) generateDriverSpec(cluster cephv1.CephCluster) (csiopv1.D
 		return csiopv1.DriverSpec{}, errors.Wrapf(err, "failed to create ceph-CSI operator config ImageSetConfigmap for CR %s", opConfigCRName)
 	}
 
+	// Determine the right pod labels based on driver type
+	var podLabels map[string]string
+	switch driverType {
+	case "cephfs":
```

Comment:
instead of string its better to define const types for it to avoid coding errors

Reply:
Not needed anymore